packages/fxa-shared/guards/AdminPanelGuard.ts

```diff
@@ -68,28 +86,36 @@ const defaultAdminPanelPermissions: Permissions = {
     name: 'View Active Sessions',
     level: PermissionLevel.Support,
   },
+  [AdminPanelFeature.LinkedAccounts]: {
+    name: 'View Linked Accounts',
+    level: PermissionLevel.Support,
+  },
   [AdminPanelFeature.ClearEmailBounces]: {
     name: 'Clear Email Bounces',
     level: PermissionLevel.Support,
   },
-  [AdminPanelFeature.DisableAccounts]: {
+  [AdminPanelFeature.DisableAccount]: {
     name: 'Disable Accounts',
     level: PermissionLevel.Admin,
   },
   [AdminPanelFeature.SiteStatus]: {
     name: 'Site Status',
     level: PermissionLevel.Support,
   },
-  [AdminPanelFeature.UnVerifyAccounts]: {
-    name: 'Unverify Accounts',
+  [AdminPanelFeature.UnverifyEmail]: {
+    name: 'Unverify Email Address',
+    level: PermissionLevel.Admin,
+  },
+  [AdminPanelFeature.UnlinkAccount]: {
+    name: 'Unlink Account',
     level: PermissionLevel.Admin,
   },
 };
 
 /** Setup configured guard for admin panel */
 class AdminPanelGuard extends Guard<AdminPanelFeature, AdminPanelGroup> {
   constructor() {
-    super(defaultAdminPanelPermissions, defaultAdminPanelGroups);
+    super(defaultAdminPanelPermissions, adminPanelGroups);
```

Comment:
Might be a hair more consistent to also change the name of `defaultAdminPanelPermissions` to `adminPanelPermissions` if you prefer to remove "default".